src/tokens/zIndex.js

```diff
@@ -0,0 +1,30 @@
+const light = (core) => ({
+  // Global
+  SendToBack: core.ZIndex.SendToBack,
+  Default: core.ZIndex.Base,
+  Hover: core.ZIndex.Hover,
+  Active: core.ZIndex.Active,
+  Focus: core.ZIndex.Focus,
+  Overlay: core.ZIndex.Overlay1,
+  Sticky: core.ZIndex.Overlay2,
+  Fixed: core.ZIndex.Overlay3,
+  // Date Picker
+  DatePicker: core.ZIndex.Overlay1,
+  // Drawer
+  Drawer: core.ZIndex.Overlay5,
+  // Dropdown Menu
+  DropdownMenu: core.ZIndex.Overlay1,
+  // Message Box
+  MessageBox: core.ZIndex.Overlay3,
+  // Modal
+  Modal: core.ZIndex.Overlay5,
+  // Tooltip
+  Tooltip: core.ZIndex.Overlay7,
+})
+
+const dark = (core) => ({})
```

Comment:
I'm not sure what error you are having when building, but the dark tokens are going to be empty unless you spread the light tokens in like we do with the Spacing tokens like this:
```
const dark = (core) => ({
  ...light(core),
})
```

Reply:
also, make sure to run `npm build:docs` and `npm run json`

Reply:
Fixed that @jgerlach

Reply:
Looks good to me. I'm going to approve it. You might want Cameron to take a look before you publish it.